src/app/components/Forms/RegisterForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import Button from '../Button/Button'
+import FormInput from '../FormInput/FormInput'
+import { BiHide, BiShow } from 'react-icons/bi'
+
+type LoginFormProps = {
+  onSubmit: (payload: {
+    username: string
+    email: string
+    password: string
+  }) => void
+  onCancel: () => void
+}
+
+export default function LoginForm({
+  onSubmit,
```

Comment:
Why is it called LoginFormProps and LoginForm when it's actually the RegisterForm? ;)